Mathlib/Algebra/Polynomial/AlgebraMap.lean

```diff
@@ -532,6 +554,19 @@ theorem aevalTower_ofId : aevalTower (Algebra.ofId S A') = aeval := by
 
 end aevalTower
 
+open LinearMap TensorProduct in
+lemma X_pow_mul_rTensor_monomial [CommSemiring S] [Algebra R S] {N : Type*}
```

Comment:
```suggestion
lemma X_pow_smul_rTensor_monomial [CommSemiring S] [Algebra R S] {N : Type*}
```
Also I think this lemma looks a bit weird and it could benefit from a docstring describing what it is doing.